sdk/python/kfp/deprecated/notebook/_magic.py

```diff
@@ -14,5 +14,3 @@
 
 import os
 import tempfile
-
-from kfp.deprecated.compiler import build_docker_image
```

Comment:
Can you explain what this file does? I'm curious how we know we can safely remove this import.

Reply:
This is a deprecated item that triggers other test failures :) We could safely remove this, since it was deprecated in v1.